src/components/LoadingSpinner.vue

```diff
@@ -3,36 +3,36 @@ defineProps({
   fillWhite: {
     type: Boolean,
     default: false
+  },
+  small: {
+    type: Boolean,
+    default: false
   }
 });
 </script>
 
 <template>
-  <BaseLoading class="loading">
-    <span>
-      <span>
-        <svg
-          xmlns="http://www.w3.org/2000/svg"
-          width="50"
-          height="50"
-          viewBox="0 0 50 50"
-        >
-          <path
-            :class="{ '!fill-white': fillWhite }"
-            d="M25,5A20.14,20.14,0,0,1,45,22.88a2.51,2.51,0,0,0,2.49,2.26h0A2.52,2.52,0,0,0,50,22.33a25.14,25.14,0,0,0-50,0,2.52,2.52,0,0,0,2.5,2.81h0A2.51,2.51,0,0,0,5,22.88,20.14,20.14,0,0,1,25,5Z"
-          >
-            <animateTransform
-              attributeName="transform"
-              type="rotate"
-              from="0 25 25"
-              to="360 25 25"
-              dur="0.5s"
-              repeatCount="indefinite"
-            />
-          </path>
-        </svg>
-      </span>
-    </span>
+  <BaseLoading class="loading" :class="{ small: small }">
```

Comment:
Sugaring the syntax a bit.

```suggestion
  <BaseLoading class="loading" :class="{ small }">
```


Or... the readability version (renaming the prop):

```suggestion
  <BaseLoading class="loading" :class="{ 'small': isSmall }">
```

Or... skip this and go on with your life. (probably most reasonable option :D)